source/declarations.tex

```diff
@@ -1771,9 +1771,11 @@
 
 \pnum
 The type of a \grammarterm{parameter-declaration} of a
-function declaration\iref{dcl.fct},
-\grammarterm{lambda-expression}\iref{expr.prim.lambda}, or
-\grammarterm{template-parameter}\iref{temp.param}
+\begin{itemize}
+  \item function declaration\iref{dcl.fct}, or
```

Comment:
one "or" too many.  The original text did not have an "or" here.

Remember, we're just adding bullets and not otherwise changing the sentence structure.

Reply:
Ok, sure, done